<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab11_TNR</class>
 <widget class="QMainWindow" name="Lab11_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab11_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="n_TNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_TNR">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>starting from m:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Condititon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number must be</string>
    </property>
   </widget>
   <widget class="QComboBox" name="condition1_TNR">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="conditionNum_TNR">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="output1_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Output container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output2_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Output container #2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="out_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rand1Btn_tnr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill container #1 rand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputS_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Output second stack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputQ_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Output second queue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randSBtn_tnr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill second stack rand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_tnr">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete from container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sto1Btn_tnr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Push from second stack to container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort1Btn_tnr">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort2Btn_tnr">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort container #2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getNum_tnr">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Get number from container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill2from1Btn_tnr">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill container #2 from container #1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortWay_tnr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="mergeBtn_tnr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge containers into second queue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countQBtn_tnr">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Count second queue -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_tnr">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_tnr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear1_tnr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear container #1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear2_tnr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear container #2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToSBtn_tnr">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="addToSVal_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addToQBtn_tnr">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="addToQVal_tnr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
